lib/model/classycat_response.py

```diff
@@ -3,11 +3,12 @@
 
 
 class ClassyCatResponse(BaseModel):
+    # TODO: replase with the presto repose class?
```

Comment:
is this to replace `ClassyCatResponse` with presto response class? can you elaborate a bit more on what this could be like in practice please?

Reply:
lol this comment may beat my personal best for number of spelling errors per line!

I think I was meaning that the response message and list of items that ClassyCat was defining are now part of the Presto`Response class in schemas.py and enforced for all models.  So ClassyCat can import and extend that class and the ResponseItem class (into the classy schemas files) to add the ClassyCat specific fields